google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionImpl.java

```diff
@@ -133,17 +133,6 @@ enum BatchMode {
     DML
   }
 
-  /**
```

Comment:
This was dead code. Only the class in the public `Connection` interface was used.